Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1293,37 +1240,13 @@ PhysicalParticleContainer::Evolve (int lev,
                     }
                     
                     // Field gather for particles in gather buffers
-#ifdef WARPX_RZ
-                    
```

Comment:
Could we also get rid of lines 1179 and 1180:
```
                    const std::array<Real,3>& cxyzmin_grid = WarpX::LowerCorner(cbox, lev-1);
                    const int* cixyzmin_grid = cbox.loVect();
```
as you did for the field gather on fine patch?